src/Microsoft.TestPlatform.Common/Utilities/AssemblyType.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.Common.Utilities
+{
+    public enum AssemblyType
+    {
+        NativeAndManaged,
```

Comment:
do we need NativeAndManaged, can't we ask user specify Category attribute two time?

Reply:
Category attribute is not supported multiple times.
Adapter which supports both native and managed will not specify category attribute.